locales/en/index.yml

```diff
@@ -3315,11 +3315,11 @@ features:
     issuance:
       credentialAuth:
         title: "{{credentialName}}: dati necessari"
-        subtitle: "Saranno condivisi con **{{organization}}** per il rilascio della credenziale"
+        subtitle: "Saranno condivisi con **{{organization}}** per il rilascio della versione digitale del documento."
         requiredClaims: Dati richiesti
         disclaimer:
           0: I tuoi dati sono al sicuro e saranno trattati solo per le finalità descritte in informativa Privacy.
-          1: I dati saranno condivisi solo per il tempo necessario al rilascio della credenziale.
+          1: I dati saranno condivisi solo per il tempo necessario al rilascio della versione digitale del documento.
         tos: "Premendo **Continua** dichiari di aver letto e compreso l’[Informativa Privacy](https://io.italia.it/informativa-ipzs-sperimentazione/)."
```

Comment:
Is this privacy url (https://io.italia.it/informativa-ipzs-sperimentazione) valid? I get 404.

Reply:
the URL in env file is valid for me https://io.italia.it/informativa-sperimentazione

Reply:
The correct one is https://io.italia.it/informativa-ipzs Thanks for the catch!